src/commands/update.ts

```diff
@@ -0,0 +1,46 @@
+import { ExtendedGluegunToolbox } from '../interfaces/extended-gluegun-toolbox'
+import { GluegunCommand } from 'gluegun'
+
+/**
+ * Update @lenne.tech/cli
+ */
+const NewCommand: GluegunCommand = {
+  name: 'update',
+  alias: ['up'],
+  description: 'Update @lenne.tech/cli',
+  hidden: false,
+  run: async (toolbox: ExtendedGluegunToolbox) => {
+    // Retrieve the tools we need
+    const {
+      helper,
+      print: { info, spin, success },
+      system: { run, startTimer }
+    } = toolbox
+
+    // Start timer
+    const timer = startTimer()
+
+    // Update
+    const updateSpin = spin(`Update @lenne.tech/cli`)
+    await run('npm install -g @lenne.tech/cli')
+    updateSpin.succeed()
+
+    // Check new version
+    const versionSpin = spin(`Get current version from @lenne.tech/cli`)
+    const version = await run('lt version')
```

Comment:
Sometimes the result of a run is formatted. For example, line breaks could be included. Therefore you should always use helper.trim to remove at least unwanted line breaks:

```
const version = helper.trim(await run ('lt version'))
```